tests/integrations/docker/test_verify_docker.py

```diff
@@ -113,3 +120,60 @@ def test_docker_unknown_version(test_command, capsys):
     output = capsys.readouterr()
     assert '** WARNING: Unable to determine the version of Docker' in output.out
     assert output.err == ''
+
+
+def test_docker_exists_but_process_lacks_permission_to_use_it(test_command, capsys):
```

Comment:
Given that there are multiple error messages that can trigger this, we should test all of them. This is a good candidate for a parameterized test.

Reply:
I parametrized `test_docker_exists_but_is_not_running` based on this feedback - right now I only have one error string that triggers the failure state in this test.  I may be misunderstanding what you are referring to here otherwise.  
Would you instead prefer that 

`test_docker_exists_but_process_lacks_permission_to_use_it`
`test_docker_exists_but_is_not_running`
`test_docker_exists_but_unknown_error_when_running_command`

[the 3 failure modes of `_verify_docker_can_run`, of which the 2nd one has 2 strings it checks]

are all 1 parametrized test?

Reply:
No - I was referring to `docker_exists_but_is_not_running`, but I attached the comment to the wrong test title. There are two error messages that can cause that branch to execute, but you were only testing one of them. Sorry for the misdirection on that one.